tox.ini

```diff
@@ -0,0 +1,37 @@
+[tox]
+minversion = 2.0
+envlist = py34,py27,pypy,pep8
+skipsdist = True
+
+[testenv]
+usedevelop = True
+install_command = pip install -U {opts} {packages}
+setenv = VIRTUAL_ENV={envdir}
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+
+[testenv:pep8]
+commands = flake8 {posargs}
+
+[testenv:venv]
+commands = {posargs}
+
+[testenv:cover]
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+commands =
+    py.test --cov-report= --cov=python_gdeploy python_gdeploy/tests
```

Comment:
Check node-agent develop branch for new code coverage tool changes to tox.ini and requirements etc

https://github.com/Tendrl/node-agent/blob/develop/tox.ini#L30